src/app/components/SearchResultsPage/SearchResultsPage.jsx

```diff
@@ -40,7 +40,7 @@ const SearchResultsPage = (props, context) => {
   const createAPIQuery = basicQuery(props);
   const h1searchKeywordsLabel = searchKeywords ? `for ${searchKeywords}` : '';
   const h1pageLabel = totalPages ? `page ${page} of ${totalPages}` : '';
-  const h1Label = `Search results ${h1searchKeywordsLabel} ${h1pageLabel}`;
+  const h2Label = `Search results ${h1searchKeywordsLabel} ${h1pageLabel}`;
```

Comment:
let's change the two variables's names too? Maybe we can skip `h1` so we don't have to change it again time if HTML updates again.